rasa/utils/common.py

```diff
@@ -148,6 +149,15 @@ def update_sanic_log_level(log_file: Optional[Text] = None) -> None:
         logger.addHandler(file_handler)
         error_logger.addHandler(file_handler)
         access_logger.addHandler(file_handler)
+    if use_syslog:
+        formatter = logging.Formatter("%(asctime)s [%(levelname)-5.5s] [%(process)d]"
+                                      " %(message)s")
+        syslog_handler = logging.handlers.SysLogHandler(address='/dev/log')
```

Comment:
```suggestion
        syslog_handler = logging.handlers.SysLogHandler(address=log_file)
```

Reply:
This would totally break the possibility to log in syslog
The idea is that you can log in a log file and in syslog in parallel.

Reply:
I see what you meant about cross plateform subject. 
I'm making a change to use the default communication with the syslog server. I'll make a new request once it's done.